spec.html

```diff
@@ -32966,7 +32972,7 @@ <h1>
           GetNamedTimeZoneOffsetNanoseconds (
             _timeZoneIdentifier_: a String,
             _epochNanoseconds_: a BigInt,
-          ): an integer
+          ): an integer in the exclusive interval from -8.64 × 10<sup>13</sup> to 8.64 × 10<sup>13</sup>
```

Comment:
We actually don't define the term "exclusive interval". We should add a `dfn` for it.